TopicPage.js

```diff
@@ -327,8 +327,14 @@ const TopicPage = ({ topic, onBack, openNav, openTopic, showToast, openRef, setT
   const [parashaData, setParashaData] = useState(null);
   const [query, setQuery] = useState(null);
   const [portal, setPortal] = useState(null);
+  const isAuthor = topicData?.subclass === 'author';
+  const hasWorks = isAuthor && topicData?.indexes?.length;
   const tabs = [];
-  if (!!topicData && !!topicData.textRefs.length) { tabs.push({text: strings.sources, id: 'sources'}); }
+  if (hasWorks) {
```

Comment:
now that sheets are out of the app, it seems odd to keep using tabs when there's only one tab.